test/queue_IO_check.cxx

```diff
@@ -52,12 +52,12 @@ auto timeout = std::chrono::milliseconds(100); ///< Queue's timeout
  */
 std::unique_ptr<dunedaq::appfwk::Queue<int>> queue = nullptr;
 
-constexpr int nelements = 100; ///< Number of elements to push to the Queue (total)
+int nelements = 10000000; ///< Number of elements to push to the Queue (total)
```

Comment:
Is there a reason for this to be so high? This executable takes ~30 seconds to run now. Maybe 1,000,000?

Reply:
Maybe a better comment to make would be to remove some of the defaults (queue_type?) and force the user to specify things via the command-line...But that could also be a separate Issue.

Reply:
Happy to make it 1 million.

Reply:
Changed it to 1 million.